src/V8JsiRuntime.cpp

```diff
@@ -1,1628 +1,1630 @@
-
-    napi_value nstr = v8impl::JsValueFromV8LocalValue(str_maybe.ToLocalChecked());
-    auto finalize = [](napi_env env, void *finalize_data, void *finalize_hint) {
-      NapiUniqueString *uniqueString = static_cast<NapiUniqueString *>(finalize_data);
-      V8Runtime *runtime = static_cast<V8Runtime *>(finalize_hint);
-      auto it = runtime->unique_strings_.find(uniqueString->GetView());
-      if (it != runtime->unique_strings_.end()) {
-        runtime->unique_strings_.erase(it);
-      }
-    };
-    STATUS_CALL(napi_ext_create_reference_with_data(env, nstr, uniqueString.get(), finalize, this, &ref));
-    uniqueString->SetRef(ref);
-    unique_strings_[uniqueString->GetView()] = std::move(uniqueString);
-  }
-
-  *result = ref;
-  return napi_clear_last_error(env);
-}
-
-bool V8Runtime::IsEnvDeleted() noexcept {
-  return is_env_deleted_;
-}
-
-void V8Runtime::SetIsEnvDeleted() noexcept {
-  is_env_deleted_ = true;
-}
-
-//=============================================================================
-// StringViewHash implementation
-//=============================================================================
-
-size_t StringViewHash::operator()(napijsi::string_view view) const noexcept {
-  return s_classic_collate.hash(view.begin(), view.end());
-}
-
-/*static*/ const std::collate<char> &StringViewHash::s_classic_collate =
-    std::use_facet<std::collate<char>>(std::locale::classic());
-
-//=============================================================================
-// NapiUniqueString implementation
-//=============================================================================
-
-NapiUniqueString::NapiUniqueString(napi_env env, std::string value) noexcept : env_{env}, value_{std::move(value)} {}
-
-NapiUniqueString::~NapiUniqueString() noexcept {}
-
-napijsi::string_view NapiUniqueString::GetView() const noexcept {
-  return napijsi::string_view{value_};
-}
-
-napi_ext_ref NapiUniqueString::GetRef() const noexcept {
-  return string_ref_;
-}
-
-void NapiUniqueString::SetRef(napi_ext_ref ref) noexcept {
-  string_ref_ = ref;
-}
-
-//=============================================================================
-// Make V8 JSI runtime
-//=============================================================================
-
-std::unique_ptr<jsi::Runtime> makeV8Runtime(V8RuntimeArgs &&args) {
-  return std::make_unique<V8Runtime>(std::move(args));
-}
-
-std::unique_ptr<jsi::Runtime> makeV8Runtime() {
-  return std::make_unique<V8Runtime>(V8RuntimeArgs());
-}
-
-#if defined(_WIN32) && defined(V8JSI_ENABLE_INSPECTOR)
-void openInspector(jsi::Runtime& runtime) {
-  V8Runtime& v8Runtime = reinterpret_cast<V8Runtime&>(runtime);
-  std::shared_ptr<inspector::Agent> inspector_agent =
-      v8Runtime.getInspectorAgent();
-  if (inspector_agent) {
-    inspector_agent->start();
-  }
-}
-
-void openInspectors_toberemoved() { inspector::Agent::startAll(); }
-#endif
-
-} // namespace v8runtime
+// Copyright (c) Microsoft Corporation.

```

Comment:
What changed?

Line endings?

Reply:
Diffed manually.

Apparently, this version only introduces this line:
```
host_object_lifetime_tracker_list_.clear();
```

Reply:
Yes, that's the only difference. GitHub is being weird, but CodeFlow will load the diff correctly.